src/main/java/io/ebeaninternal/server/core/DefaultBeanLoader.java

```diff
@@ -288,7 +288,7 @@ private void refreshBeanInternal(EntityBean bean, SpiQuery.Mode mode, int embedd
     }
 
     if (embeddedOwnerIndex > -1) {
-      query.select("id," + ebi.getProperty(embeddedOwnerIndex));
```

Comment:
Test case first ... yes.  This change is wrong though ... it should just not include the id property at all.